gen/generator.go

```diff
@@ -1403,26 +1761,37 @@ func setDocumentationData(m *Model, definitions Definitions) {
 	} else {
 		for _, resourceDetails := range resourcesFound {
 			m.DocumentationParentDns = append(m.DocumentationParentDns, fmt.Sprintf("[%s_%s](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/%s) (%s)", providerName, resourceDetails[0], resourceDetails[0], GetDevnetDocForClass(resourceDetails[1])))
+
 		}
 	}
 
 	if len(resourcesNotFound) != 0 {
```

Comment:
This was done to display classes that could be used in the docs for which there was not resource yet defined. So it would add to 20 (which is the mac in the definition file) when more classes are possible to be configured but the resources for it have not been defined.

When this is done the math part is not needed. 
```suggestion
	if len(resourcesNotFound) != 0 && len(resourcesFound) < docsParentDnAmount {
```